index.js

```diff
@@ -12,9 +15,26 @@ const sequelizeErrorParser = require('./app/services/sequelizeErrorParser');
 
 const port = process.env.PORT;
 const app = express();
+const storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    const destFolder = req.path.split('/')[2];
+    cb(null, `${__dirname}/files/${destFolder}`);
+  },
+  filename: function (req, file, cb) {
+    crypto.pseudoRandomBytes(16, function (err, raw) {
+      if (err) {
+        return cb(err);
+      }
+
+      cb(null, `${raw.toString('hex')}-${Date.now()}${path.extname(file.originalname)}`);
```

Comment:
I recommend you to use https://www.npmjs.com/package/uuid. It is the best way to generate unique values.